recipes/hfsm2/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(hfsm2 REQUIRED CONFIG)
+
+add_executable(example example.cpp)
+target_link_libraries(example PRIVATE hfsm2::hfsm2)
+set_property(TARGET example PROPERTY CXX_STANDARD 11)
```

```suggestion
set_property(TARGET example PROPERTY CXX_STANDARD 11)

```

Missing empty EOL. To the CI, it's an error.